critiquebrainz/db/review_test.py

```diff
@@ -31,6 +31,7 @@ def test_review_creation(self):
             entity_id="e7aad618-fa86-3983-9e77-405e21796eca",
             entity_type="release_group",
             text="Testing",
+            rating=100,
```

Comment:
You added rating everywhere, but is it necessary? Would it be better to test different kinds of reviews that we'll be able to have now (with rating, without rating, with both text and rating)?